content/general/limits.textile

```diff
@@ -114,6 +130,33 @@ Message counts and rates are the number of "messages":/glossary#messages publish
 
 If the message rate is exceeded, a "global instantaneous rate limit":#global-rates will be applied to message delivery and publishing. The hourly and total message limits are "time-based":#time. If the hard limit is hit for those, message delivery and publishing will be blocked until the following hour or month.
 
+h4(#message-notifications). Message notifications
+
+Message limit notifications are split into hourly, monthly and message rate notifications. 
+
+h5(#message-hourly-notifications). Hourly notifications
+
+There are three types of hourly message limit notifications you may receive:
+
+* @messages.hourly.warning.count@ - when you reach 80% of your pre-paid quota for the number of messages.
+* @messages.hourly.soft.count@ - when your pre-paid quota for the number of messages has been exceeded.
```

Comment:
One minor point is that the descriptions for these 2 notifications aren't consistent with the others.

Reply:
Fixed.